cvat/apps/plugins/views.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from django.apps import apps
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+
+# Create your views here.
+
+class Plugins(viewsets.ViewSet):
```

Comment:
Let's implement it not as a separate app but as a part of the engine app (/api/v1/server/plugins). In this case the feature is so tiny feature that I don't think we need a separate django app.